manuscript/build-container-image/kbld.md

```diff
@@ -13,6 +22,8 @@ TODO:
 ## Do
 
 ```bash
+
+# should we use docker or CNBs?
```

Comment:
I thought to use Dockerfile mostly to distinguish it a bit more from Buildpacks. No preference though so we can change that.

Reply:
We use Dockerfile in the Lima solution, yes?  Is it different since it is backed by nerdctl?  

I'd prefer to use CNBs for this one if it is all the same to you
( ꈍᴗꈍ)

Reply:
Sounds great. Do you want to give it a shot at changing the Gist (if there are any changes) or should I?

Reply:
I just did it!